### ISSUE TITLE
`scheduleDidStopChangingEvent is not a function` only on Atom 1.19

### ISSUE THREAD

**Issue by marcoms** [TypeStrong/atom-typescript#1294]
Probably similar to #1273, this only happens on the 1.19 beta release, and causes autocompletion to break.

```
Uncaught (in promise) TypeError: this.buffer.scheduleDidStopChangingEvent is not a function
    at TypescriptBuffer.<anonymous> (typescriptBuffer.ts:61)
    at Generator.next (<anonymous>)
    at /home/marcoms/.atom/packages/atom-typescript/node_modules/tslib/tslib.js:91
    at Object.__awaiter (/home/marcoms/.atom/packages/atom-typescript/node_modules/tslib/tslib.js:87)
    at TypescriptBuffer.flush (typescriptBuffer.ts:56)
    at AutocompleteProvider.<anonymous> (autoCompleteProvider.ts:126)
    at Generator.next (<anonymous>)
    at fulfilled (/home/marcoms/.atom/packages/atom-typescript/node_modules/tslib/tslib.js:88)
```

**Comment by lierdakil** [comment]
Should probably use `emitDidStopChangingEvent` now. Here's the relevant change: https://github.com/atom/text-buffer/commit/4e55ddfcbd2ba59f82d759d036cbd864c34c587c#diff-13f48d3dd70d1cba1b533a8cc3f13416

**Comment by alexeden** [comment]
Installed Atom 1.19 today, TS autocomplete no longer works and the package emits the same error mentioned above when I press a key in the editor:
```
Uncaught (in promise) TypeError: this.buffer.scheduleDidStopChangingEvent is not a function
    at TypescriptBuffer.<anonymous> (typescriptBuffer.ts:61)
    at Generator.next (<anonymous>)
```

**Comment by wolfdenoir** [comment]
Hi, what is the status of this case? I am also experiencing the same problem where TS autocomplete does not work. I have the same error as well:

`Uncaught (in promise) TypeError: this.buffer.scheduleDidStopChangingEvent is not a function
    at TypescriptBuffer.<anonymous>`

I am running Atom 1.19.1

**Comment by lierdakil** [comment]
#1310

**Comment by lierdakil** [comment]
Should be fixed in 11.0.7
